DataFormats/OnlineMetaData/interface/DCSRecord.h

```diff
@@ -0,0 +1,75 @@
+#ifndef DATAFORMATS_ONLINEMETADATA_DCSRECORD_H
+#define DATAFORMATS_ONLINEMETADATA_DCSRECORD_H
+
+//---------------------------------------------------------------------------
+//!  \class DCSRecord
+//!  \brief Class to contain DCS information from soft FED 1022
+//!
+//!  \author Remi Mommsen - Fermilab
+//---------------------------------------------------------------------------
+
+
+#include <cstdint>
+#include <string>
+#include <vector>
+
+#include "DataFormats/OnlineMetaData/interface/OnlineMetaDataRaw.h"
+#include "DataFormats/Provenance/interface/Timestamp.h"
+
+
+class DCSRecord
+{
+public:
+
+  enum Partition {
+    EBp,EBm,EEp,EEm,HBHEa,HBHEb,HBHEc,HF,HO,
+    RPC,DT0,DTp,DTm,CSCp,CSCm,CASTOR,ZDC,
+    TIBTID,TOB,TECp,TECm,BPIX,FPIX,ESp,ESm,
+    Last
+  };
+
+  DCSRecord();
+  DCSRecord(const onlineMetaData::DCS_v1&);
+  virtual ~DCSRecord();
+
+  // Return the time of the last change
+  edm::Timestamp getTimestamp() const { return timestamp_; }
```

Comment:
return ```const edm::Timestamp &``` here. No need to copy.
The cost is probably the same now (Timestamp is currently made of two uints)

Reply:
here and for other methods in this PR, ``` get ``` prefix seems unnecessary.

Reply:
Okay, will do.